.appveyor.build.sh

```diff
@@ -5,10 +5,10 @@ JRE8_URL='https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jd
 JRE8_VERSION='AdoptOpenJDK-8u242b08'
 JRE11_URL='https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.6%2B10/OpenJDK11U-jre_x64_linux_hotspot_11.0.6_10.tar.gz'
 JRE11_VERSION='AdoptOpenJDK-11.0.6_10'
-JRE13_URL='https://github.com/AdoptOpenJDK/openjdk13-binaries/releases/download/jdk-13.0.2%2B8/OpenJDK13U-jdk_x64_linux_hotspot_13.0.2_8.tar.gz'
-JRE13_VERSION='AdoptOpenJDK-13.0.2_8'
-BUILD_JDK_URL="$JRE13_URL"
-BUILD_JDK_VERSION="$JRE13_VERSION"
+JRE14_URL='https://github.com/AdoptOpenJDK/openjdk14-binaries/releases/download/jdk-14%2B36/OpenJDK14U-jre_x64_linux_hotspot_14_36.tar.gz'
```

Comment:
This URL looks like it's for a JRE not a JDK so won't be able to build

Reply:
Ah oops, let me fix this. Thank you!